data/test_analytics_json_fields_detail.yaml

```diff
@@ -0,0 +1,35 @@
+fields:
+- name: method
+  required: false
```

Comment:
I'm not sure if we can do conditional 'required' attributes. 

I've noted in the 'desc' attribute that the attribute is required if the parent object has a specific attribute.